Sources/TuistSupport/Constants.swift

```diff
@@ -54,7 +54,7 @@ public enum Constants {
         public static let verbose = "TUIST_VERBOSE"
         public static let colouredOutput = "TUIST_COLOURED_OUTPUT"
         public static let versionsDirectory = "TUIST_VERSIONS_DIRECTORY"
-        public static let cacheDirectory = "TUIST_CACHE_DIRECTORY"
+        public static let forceConfigCacheDirectory = "TUIST_FORCE_CONFIG_CACHE_DIRECTORY"
```

Comment:
I'd mark this deprecated and add a link to the issue, so it's obvious that this will be used only temporarily 'till we fix it

Reply:
done